xds/src/main/java/io/grpc/xds/ClientXdsClient.java

```diff
@@ -123,6 +130,10 @@
   static boolean enableFaultInjection =
       Strings.isNullOrEmpty(System.getenv("GRPC_XDS_EXPERIMENTAL_FAULT_INJECTION"))
           || Boolean.parseBoolean(System.getenv("GRPC_XDS_EXPERIMENTAL_FAULT_INJECTION"));
+  @VisibleForTesting
+  static boolean enableRetry =
+      Strings.isNullOrEmpty(System.getenv("GRPC_XDS_EXPERIMENTAL_RETRY"))
```

Comment:
Why is this on by default?

Reply:
Oops. My mistake.

Reply:
The new code inverts the boolean provided on the command line, which still seems wrong.

Reply:
:scream: :sob: